### DISCUSSION TITLE
在线播放视频倍速播放的帧率

### DISCUSSION THREAD

**Discussion by jiang-yuan** [hooke007/mpv_PlayKit#270] [Q&A 问答]
在MPV播放器里面2倍速播放是直接帧率翻倍
我想了解下在线视频是不是这个原理,还是24帧的视频倍速了变成抽帧还是24帧每秒,而不是48帧每秒?比如b站腾讯视频之类的网站
因为在线播放的视频信息量一般不大,我都是倍速看的
有没有懂的兄弟讲讲?

**Comment by natural-harmonia-gropius** [comment]
看浏览器的实现
chrome用的ffmpeg，是帧率翻倍
edge有ab测试可能会调用media foundation，目测也是帧率翻倍，但是卡顿时相比ffmpeg更积极地丢帧

**Comment by hooke007** [comment]
你这脑洞大了，用抽帧的方式进行加速，处理复杂不说，还破坏画面的连贯性。

**Reply by jiang-yuan** [reply]
意思说我只要在线视频都是2倍速以上看的,其实不需要补帧了

**Reply by hooke007** [reply]
我回答的就是你的问题，甚至可以说没有播放器（至少我没见过）会用抽帧进行倍速
而且准确来讲源帧率并没有改变，只是时间轴的移动速度变了。和你按住时间轴随意拖动本质上一样，帧率并没有改变

**Reply by 2575757** [reply]
B站安卓端就是抽帧的，倍速后明显变卡，网页端丝滑

**Reply by hooke007** [reply]
我不卡，你确定不是你手机的bug导致的跳帧？
抽帧和跳帧不是一个概念
